src/lib/packages/utils.ts

```diff
@@ -168,14 +168,24 @@ export function addNewColorPicker(attribute: string): void {
 
   // create color pickter
   function createNewColor(): Node {
+    const containerElement = document.createElement('div');
+    containerElement.className = 'clr-field';
+
+    const buttonElement = document.createElement('button');
+    buttonElement.setAttribute('type', 'button');
+    buttonElement.setAttribute('aria-labelledby', 'clr-open-label');
+
     const newColorCreated = document.createElement('input');
     newColorCreated.setAttribute('type', 'text');
     newColorCreated.setAttribute('data-coloris', '');
     newColorCreated.placeholder = 'Tap to pick a color';
     newColorCreated.id = `${attribute}-color${colorNumber}`;
     newColorCreated.className = `${attribute}-inputs`;
 
-    return newColorCreated;
+    containerElement.appendChild(buttonElement);
+    containerElement.appendChild(newColorCreated);
+
+    return containerElement;
```

Comment:
Part of [this](https://github.com/Dun-sin/Code-Magic/pull/497#discussion_r1361062020). Since you asked not to change the UI as now, I would revert back to the previous changes in my next commit.